### DISCUSSION TITLE
Saving instrument patches

### DISCUSSION THREAD

**Discussion by maks** [bangcorrupt/hacktribe#82] [Ideas]
More synth than sampler related feature idea: one thing I'd really love to be able to do is save synth settings (oscillator, edit value, EG settings etc) to a name and then be able to select to use it just like with a factory or user sample. If they could be exported & imported that could open up a whole new aspect for the community around electribes which I've seen for other grooveboxes and synths which do have the functionality.

Of course I might be missing something and this could already be possible with the std midi implementation?

**Comment by bangcorrupt** [comment]
> possible with the std midi implementation

Yes, this would be possible by slicing up the pattern file.  The patch settings are in a block before each sequence.  I'll add a script at some point.

**Reply by maks** [reply]
That would be awesome to have in a script! Thanks!

**Comment by gradientledge** [comment]
awesome feature implement, i've been using the resampling feature to save patches, but this would be awesome implementation.

**Comment by ptowns** [comment]
I just copy the part to a pattern labeled "patches" and then clear the sequence after. You can copy the patch back into another pattern then.

**Reply by maks** [reply]
Yes, I've been doing that too, but it would be so much nicer to be able to deal with them as named instruments the way you do with built-in and user samples.
